presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControlModule.java

```diff
@@ -42,7 +50,7 @@ public ConnectorAccessControl getConnectorAccessControl(FileBasedAccessControlCo
         if (config.getRefreshPeriod() != null) {
             return ForwardingConnectorAccessControl.of(memoizeWithExpiration(
                     () -> {
-                        log.info("Refreshing system access control from %s", config.getConfigFile());
+                        log.info("Refreshing %s access control from %s", catalogName, config.getConfigFile());
```

Comment:
Maybe use quotes or brackets around the catalog name

Reply:
catalogs rarely have non-whitespace so it's probably not needed (im fine either way). But would be good to say we're mentioning catalog name here
```
Refreshing access control for catalog %s from %s
```